ports/nrf/boards/sparkfun_nrf52840_micromod/mpconfigboard.mk

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Ha Thach for Adafruit Industries
+ * Copyright (c) 2021 Chris Marc Dailey
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef _SPARKFUN_NRF52840_MICROMOD_H_
+#define _SPARKFUN_NRF52840_MICROMOD_H_
+
+#define _PINNUM(port, pin)    ((port)*32 + (pin))
+
+/*------------------------------------------------------------------*/
+/* LED
+ *------------------------------------------------------------------*/
+#define LEDS_NUMBER           1
+#define LED_PRIMARY_PIN       _PINNUM(0, 13)
+#define LED_STATE_ON          1
+
+/*------------------------------------------------------------------*/
+/* BUTTON
+ *------------------------------------------------------------------*/
+#define BUTTONS_NUMBER        2
+#define BUTTON_1              _PINNUM(0, 7)
+#define BUTTON_2              _PINNUM(0, 10)
+#define BUTTON_PULL           NRF_GPIO_PIN_PULLUP
+
+//--------------------------------------------------------------------+
+// BLE OTA
+//--------------------------------------------------------------------+
+#define BLEDIS_MANUFACTURER   "SparkFun"
+#define BLEDIS_MODEL          "MicroMod nRF52840"
+
+//--------------------------------------------------------------------+
+// USB
+//--------------------------------------------------------------------+
+#define USB_DESC_VID           0x1B4F
+#define USB_DESC_UF2_PID       0x0022
+#define USB_DESC_CDC_ONLY_PID  0x0023
+
+//------------- UF2 -------------//
+#define UF2_PRODUCT_NAME       "SparkFun MicroMod nRF52840"
+#define UF2_VOLUME_LABEL       "SFMM852BOOT"
+#define UF2_BOARD_ID           "micromod-nRF52840"
```

Comment:
This file looks like tinyusb config, not circuitpython config.

Reply:
Absolutely is, not sure how it got in here.

Reply:
So, I just used GitHub on the web to update the PIDs for the bootloader and this at the same time. (Literally was going through the edit/commit/PR flow on both sides. *Somehow*, Github managed to commit this file despite being in the other repo. Look at this:

![image](https://user-images.githubusercontent.com/602691/110546573-a7c8ca00-80fc-11eb-819d-38955b63bca1.png)

Not that it matters at all, but really crazy. Never seen that!